x/feemarket/spec/02_state_transitions.md

```diff
@@ -0,0 +1,48 @@
+<!--
+order: 2
+-->
+
+# State Transitions
+
+This document describes the state transition operations pertaining to the base gas prices.
+
+## Block Gas Used
+
+The total gas used by current block is stored in chain state at the `EndBlock` event.
+
+It's initialized to `-1` in `InitGenesis`.
```

Comment:
Would it be possible to add another .md file for how the state (kv store) looks like? In particular:
- under which key is this total gas stored in the substore? (e.g. `0x0`)
- which encoding to use for the integer?

Also see [SPEC_SPEC](https://github.com/cosmos/cosmos-sdk/blob/master/docs/spec/SPEC-SPEC.md) for more info.